contracts/SpokePool.sol

```diff
@@ -268,6 +272,61 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         );
     }
 
+    function increaseRelayFee(
```

Comment:
Should the relayer be able to use this method to actually fill the relay? If not, how does it get connected with a particular fill? Or is the idea that we would swap out the relayerFeePct for every FilledRelay that was submitted afterward?

Reply:
The relayer in this implementation would call this method separately and it would not add to the fill. We could add an additonal method `increaseRelayFeeAndFillRelay` (bad name) that would call `fillRelay` AND modify the fee WDYT?

Reply:
> relayer in this implementation would call this method separately and it would not add to the fill

How would this work? Wouldn't the subsequent fill still be using the _old_ relayer fee (since that's the only one that would match the hash)? So in that case, too much money would be sent to the recipient, no?

We don't have to do it this way, but the way I had conceptualized this was that we'd have a fill method that's identical to the existing one with two additional args: `bytes memory signature, uint64 updatedRelayerFeePct`. The signature would be verified against the provided fee, and the updated fee fee would be emitted in the `FilledRelay` event (we could also include a second event to signify that this fee was overridden or something).

Reply:
So the relay hash would stay the same, with the old relayer fee, but the event would just change? (or have an additional one emitted)

I agree wth you that `fillRelay` could be overridden with a separate interface

Reply:
Yeah, in the solution I had been thinking of, the relay hash/data would always be the same, the signature would be provided essentially as an override to the originally-set relayer fee.

Reply:
interesting let me try this

Reply:
ye, what @mrice32 describes fits with my understanding of this method. you can actually just overload the `fillRelay` method with an extra param which optionally lets you change the originally provided `relayFeePct`.

Reply:
The only problem with overloading is that overloading doesn't play nicely with [ethers like it does with web3](https://github.com/ethers-io/ethers.js/issues/407#issuecomment-458329708). Do you have a strong opinion about overloading versus creating a new function name?